modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field.js

```diff
@@ -267,12 +267,12 @@ AUI.add(
 						return Lang.String.unescapeHTML(inputNode.val());
 					},
 
-					hasFocus: function() {
+					hasFocus: function(node) {
 						var instance = this;
 
 						var container = instance.get('container');
 
-						return container.contains(document.activeElement);
+						return container.contains(node);
```

Comment:
Do you think it would be good to check if node is not undefined in case there is some old code using the function without the node parameter?

Reply:
Thank you, @linolaoj. I'll do it.